READMEへのデータベース設計の追記

OzakiRoy opened the pull request:
# WHAT
README.mdへ本アプリケーションのデータベース設計を記述する。

# WHY
データベース設計を明示しておくため。

Commit:
added Database design

Commit:
To modify typo at Database creation section

Commit:
To modify data type of groups table

hzak2xx commented on README.md:
```diff
@@ -12,6 +12,58 @@ Things you may want to cover:
+
+|Column|Type|Options|
+|------|----|-------|
+|user_name|string|null: false, unique: true|
```
nameで大丈夫です

OzakiRoy replied:
承知しました。修正しました。

hzak2xx commented on README.md:
```diff
@@ -12,6 +12,58 @@ Things you may want to cover:
+|password|string|null: false|
+
+### Association
+- has_many :groups,through: :members
```
membersのアソシエーションを書きましょう

OzakiRoy replied:
ありがとうございます、理解しました。

hzak2xx commented on README.md:
```diff
@@ -12,6 +12,58 @@ Things you may want to cover:
+|group_name|string|null: false|
+
+### Association
+- has_many :users,through: :members
```
membersのアソシエーションを書きましょう

OzakiRoy replied:
修正しました。

hzak2xx commented on README.md:
```diff
@@ -12,6 +12,58 @@ Things you may want to cover:
+
+|Column|Type|Options|
+|------|----|-------|
+|group_name|string|null: false|
```
nameで大丈夫です

OzakiRoy replied:
修正しました

hzak2xx commented on README.md:
```diff
@@ -12,6 +12,58 @@ Things you may want to cover:
 
 * Database creation
+> Prime key and timestamps are not written in the tables below. Those columns are created by default.
+------------------------
```
これより上消しましょう

OzakiRoy replied:
はい！削除しました

Rozaki0318 commented:
頂いたコメント全て反映いたしました
再度レビューお願いいたします

Commit:
To modify per mentor's comment

Changed the column name of users table and groups table.
Added the association b/w users and members table, groups and members
table.
